### PR TITLE
Support writing V3000 reactions

### PR DESCRIPTION
**Pull request by greglandrum**
We've had a parser for a while, but never got around to writing the writer.
This fixes that.

### CODE REVIEW

**Comment by greglandrum** [comment]
I'm trying to track down the win64 dll build problems.

**Comment by ptosco** [comment]
@greglandrum For some reason the GitHub web interface gave me a 404 when I tried to make a PR against your branch, so I ended up committing directly.
Including the header where `getV3000CTAB` is declared makes sure that the function is decorated with  `dllexport` and hence exported to the DLL, which otherwise misses that symbol.

**Comment by greglandrum** [comment]
Thanks @ptosco ; let’s see if that’s enough to fix it

File: Code/GraphMol/FileParsers/MolFileWriter.cpp [hunk 1]
```diff
@@ -1155,7 +1156,74 @@ void appendEnhancedStereoGroups(std::string &res, const RWMol &tmol) {
...
+  if (confId >= 0 || tmol.getNumConformers()) {
+    conf = &(tmol.getConformer(confId));
+  }
+
+  std::string res = "M  V30 BEGIN CTAB\n";
+  std::stringstream ss;
+  ss << "M  V30 COUNTS " << nAtoms << " " << nBonds << " " << nSGroups << " 0 "
+     << chiralFlag << "\n";
+  //      |
+  //      num3DConstraints (not implemented)
```
Comment (ricrogz) [inline Code/GraphMol/FileParsers/MolFileWriter.cpp:1179]: Github's formatting might be fooling me, but this doesn't look like it's pointing at the " 0 " field above anymore. We should probably update this comment to not rely on the formatting.

**Review by ricrogz — APPROVED** [review]
I'm ok with this, once the comment about "num3DConstraints" is fixed.

**Pull request merged**